bobtemplates/plone/view/views/+view_name_normalized+.py.bob

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+{{% if view_python_class %}}
+from {{{ package.dottedname }}} import _
+from Products.Five.browser import BrowserView
+
+
+class {{{ view_python_class_name }}}(BrowserView):
+
+    def __call__(self):
+        return _(u'This view works!')
```

Comment:
The view is not rendering well. I expect the plone UI with your string, not just a string.

Reply:
@MrTango For this, we need a template, isn't it?